packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -688,6 +729,14 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
       filteredEntries = filter(widget.dropdownMenuEntries, _localTextEditingController!);
     }
 
+    if (widget.enableFilter) {
```

Comment:
I think here should be:
```suggestion
    if (_enableFilter) {
```
then just remove line 728 to line 730 because that's already included in here. So the implementation would match what we said in the documentation; when we enable `enableFilter`, if the callback is not null, we will return a list which is calculated by the `filterCallback`.

Reply:
@QuncCccccc done!